src/css/components/confirm.scss

```diff
@@ -0,0 +1,38 @@
+
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.confirm {
+  width: 100%;
+}
+
+.confirm-nexto {
```

Comment:
It should be that because that would mean `to` is an element or modifier, wheresas "next to" is the modifier.